src/content/development/contribution-guide/_index.en.md

```diff
@@ -78,11 +78,12 @@ To be expanded
 
 ### Unplanned Work
 
-To be expanded
-
 #### Ongoing Maintenance
 
-To be expanded
+Tasks that relate to ongoing maintenance (version bumps, image trimmings, CI and automation) will usually be small and unplanned
+tasks that we can't plan ahead for, but we know there will always be work to do.
+Use label "maintenance" on the issue.
+In case the task has a major effect (e.g. needs more than a few days of work) then it should go through an exception process.
```

Comment:
```suggestion
In case the task has a major effect (e.g. needs more than a few days of work) then it should go through the exception process.
```

I would suggest even a single day of work; anything more than that should go through prioritisation. There might be “short-circuiting” justifications (*e.g.* security issues actually affecting Submariner) but they still need to be made explicit.